app.R

```diff
@@ -75,9 +76,28 @@ ui <- dashboardPage(
   )
 )
 
+ui <- shinymanager::secure_app(ui, enable_admin = TRUE)
+
 # Create Server Code.
 server <- function(session, input, output) {
   
+  # check_credentials directly on sqlite db
+  res_auth <- secure_server(
+    check_credentials = check_credentials(
+      file.path("credentials.sqlite"),
+      passphrase = key_get("R-shinymanager-key", "obiwankenobi")
+      # passphrase = "passphrase_wihtout_keyring"
+    )
+  )
+
+  output$auth_output <- renderPrint({
+    reactiveValuesToList(res_auth)
+  })
+  
+  observe({
+    req(!is_empty(res_auth$user))
+    print(paste("is user", res_auth$user, "an admin?", res_auth$admin))
```

Comment:
Maybe log this info to the `loggit.json` instead of printing it?

Reply:
Loggit would be better than print, agreed